contrib/mkimage-yum.sh

```diff
@@ -63,9 +68,15 @@ if [ -d /etc/yum/vars ]; then
 	cp -a /etc/yum/vars "$target"/etc/yum/
 fi
 
-yum -c "$yum_config" --installroot="$target" --releasever=/ --setopt=tsflags=nodocs \
-    --setopt=group_package_types=mandatory -y groupinstall Core
-yum -c "$yum_config" --installroot="$target" -y clean all
+if [ -f /etc/dnf/dnf.conf ] ; then 
+	dnf -c "$yum_config" --installroot="$target" --releasever=/ --setopt=tsflags=nodocs \
+		--setopt=group_package_types=mandatory -y groupinstall Core
+	dnf -c "$yum_config" --installroot="$target" -y clean all
+else
+	yum -c "$yum_config" --installroot="$target" --releasever=/ --setopt=tsflags=nodocs \
+    		--setopt=group_package_types=mandatory -y groupinstall Core
+	yum -c "$yum_config" --installroot="$target" -y clean all
```

Comment:
What I meant with my previous comment was that the two code blocks in this `if` statement are _identical_ sans the change from `yum` to `dnf`, so I'd much rather see us do this check up higher and either have a variable of `yum=dnf` and use `$yum` here, or just do `alias yum=dnf` above so that `yum` isn't invoked.

ie:
```bash
yum_config=/etc/yum.conf
if [ -f /etc/dnf/dnf.conf ] && command -v dnf &> /dev/null; then
	yum_config=/etc/dnf/dnf.conf
	alias yum=dnf
fi

# ...

yum -c "$yum_config" --installroot="$target" --releasever=/ --setopt=tsflags=nodocs \
	--setopt=group_package_types=mandatory -y groupinstall Core
yum -c "$yum_config" --installroot="$target" -y clean all
```